django/django.go

```diff
@@ -27,56 +27,39 @@ type Engine struct {
 	Templates map[string]*pongo2.Template
 }
 
-// New returns a Django render engine for Fiber
-func New(directory, extension string) *Engine {
-	engine := &Engine{
-		Engine: core.Engine{
-			Left:       "{{",
-			Right:      "}}",
-			Directory:  directory,
-			Extension:  extension,
-			LayoutName: "embed",
-			Funcmap:    make(map[string]interface{}),
-		},
-		autoEscape: true,
+// This helper function is used to avoid duplication in public constructors.
+func (e *Engine) initialize(directory, extension string, fs http.FileSystem) {
+	e.Engine.Left = "{{"
+	e.Engine.Right = "}}"
+	e.Engine.Directory = directory
+	e.Engine.Extension = extension
+	e.Engine.LayoutName = "embed"
+	e.Engine.Funcmap = make(map[string]interface{})
+	e.autoEscape = true
+	if fs != nil {
```

Comment:
won't e.Engine.FileSystem be nil with or without this check?

Reply:
Yeah, the check has 0 use. It either is nil or has a filesystem